add-ons/APPLICATION_TEMPLATE.md

```diff
@@ -0,0 +1,182 @@
+# New Project Project Application
+
+Please see the [proposals/new-project-addition-process.md](Documentation) to
+better understand the full process for submitting a new project.
+
+In short, copy this Application Template and populate the document.
+--!>
+
+
+## Background information
+
+1. Submitter Name
+-
+
+2. Submitter’s relationship to project / title
+-
+
+3. Project Name
+-
+
+4. Why this project is valuable to the Kubeflow Community
+-
+
+5. Short Description / Functionality
+-
+
+6. Website
+-
+
+7. GitHub repository
+-
+
+8. 1st Release date
+-
+
+9. Project Meeting Times
+-
+
+10. Meeting Notes (link)
+-
+
+11. Installation Documentation (provide status and link)
+-
+
+12. Application Documentation (provide status and link)
+-
+
+13. Security Profile (CVE scanning)
+-
+
+14. Ownership / Legal Profile (license type, any potential issues for CNCF)
+-
+
+15. Kubernetes native (K8s version)
+-
+
+16. AuthN/Z, Isolation mechanisms
+-
+
+17. Project Roadmap
+-
+
+18. Other Information
+-
+
+## Metrics
+
+Number of Contributors
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+Number of Users
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+GitHub Pull Requests in the last 90 days
+- [ ] 1
+- [ ] <10
+- [ ] 10+
+
+Number of Releases
+- [ ] No release
+- [ ] 1 release
+- [ ] 2+ releases
+
+Downloads
```

Comment:
Download of what ? Packages, Helm Charts, Docker images.

Reply:
I meant packages as we could get them easily here: https://www.pepy.tech/projects/feast?versions=0.*